schema.sql

```diff
@@ -39,4 +30,30 @@ CREATE TABLE vets ( id INT GENERATED ALWAYS AS IDENTITY, name VARCHAR(100), age
 
 CREATE TABLE specializations ( species_id INT, vets_id INT, FOREIGN kEY (species_id) REFERENCES species(id), FOREIGN kEY (vets_id) REFERENCES vets(id), PRIMARY kEY (species_id, vets_id));
 
-CREATE TABLE visits ( animals_id INT, vets_id INT, date_of_visit DATE, id INT GENERATED ALWAYS AS IDENTITY, FOREIGN KEY (animals_id) REFERENCES animals(id), FOREIGN KEY (vets_id) REFERENCES vets(id), PRIMARY KEY (id) );
\ No newline at end of file
+CREATE TABLE visits ( animals_id INT, vets_id INT, date_of_visit DATE, id INT GENERATED ALWAYS AS IDENTITY, FOREIGN KEY (animals_id) REFERENCES animals(id), FOREIGN KEY (vets_id) REFERENCES vets(id), PRIMARY KEY (id) );
+
+
+-- TASK 5
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+
+-- This will add 3.594.280 visits considering you have 10 animals, 4 vets, and it will use around ~87.000 timestamps (~4min approx.)
+INSERT INTO visits (animals_id, vets_id, date_of_visit) SELECT * FROM (SELECT id FROM animals) animals_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+
+-- This will add 2.500.000 owners with full_name = 'Owner <X>' and email = 'owner_<X>@email.com' (~2min approx.)
+insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
```

Comment:
- [ ] As described in the project requirements, If you decide to commit any `INSERT INTO` queries - remember to add the to the data.sql file. It is important to keep only the queries that change the database structure in the schema.sql file.